docs/getting_started.md

```diff
@@ -47,11 +47,15 @@ Here are the steps:
          keyName: app-key
    ```
 
+1. Refer to the [`Datadog Agent examples subfolder`](https://github.com/DataDog/datadog-operator/tree/main/examples/datadogagent) and select an appropriate Datadog Agent configuration file.
```

Comment:
Please use the other style of linking ([link text][#]), that is used throughout this document. Our translation process relies on all of the links being gathered at the bottom of the file.

Reply:
Hi @urseberry 
Thanks. The issue [is resolved](https://github.com/honghuac/datadog-operator/commit/018e69d22b3660261103d9991cd7f130737bec0c).